resources/views/layouts/app.blade.php

```diff
@@ -14,6 +14,10 @@
 
     <!-- Scripts -->
     <script src="{{ asset('vendor/firefly/js/app.js') }}" defer></script>
+    @if (config('firefly.features.wysiwyg'))
+    <script src="//cdn.quilljs.com/1.3.6/quill.min.js"></script>
+    <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet">
```

Comment:
Should we allow the theme to be configurable if quill offers more than just _snow_?

Reply:
I was thinking about that. There are a lot of configurability options, in terms of theming, and also the toolbar / buttons that are visible. My thought process was that this is what is provided "out of the box" and there is always the option to publish and then override the views, if one wants to customize the theme or buttons.


Another idea would be to make features => wysiwyg an array, rather than a boolean, something like this:
```php
'features' => [
        'watchers' => true,
        'wysiwyg'  => [
            'theme' => 'snow',
            'toolbar_options' => []
        ],
        'correct_posts' => true
    ],
```
 Thoughts?

Reply:
Sorry, I didn't see this, but yes, I agree with putting in a config like that.

Reply:
I would also include an `enabled` key in that array.

Reply:
I have implemented this